docs/quickstart/installation.md

```diff
@@ -38,14 +32,15 @@ python -m pip install --upgrade pip
 ## Dependencies
 
 (pytorch)=
+
 ### pytorch
 
 ::::{tab-set}
 :::{tab-item} Torch 2.0.1 with CUDA 11.8
 
 - To install 2.0.1 with CUDA 11.8:
 
-Note that if a pytorch version prior to 2.0.1 is installed, 
+Note that if a pytorch version prior to 2.0.1 is installed,
 it should be uninstalled first to avoid upgrade issues (e.g. with functorch)
```

Comment:
My understanding is that functorch may need to be explicitly uninstalled
`pip uninstall functorch`
because just uninstalling torch doesn't seem to uninstall functorch.
However, that may also impact other projects.

Reply:
I'm confused what your recommendation is here, the following line uninstalls functorch

Reply:
Updated to make more clear